model/user.go

```diff
@@ -802,6 +802,34 @@ func (u *UserPatch) SetField(fieldName string, fieldValue string) {
 	}
 }
 
+func (u *User) ShouldNotifyCRT(isMentioned bool) (bool, bool, bool) {
```

Comment:
Same as above. Let's move this inside app/notification.go and make it a private function.